admin/resources/js/components/form/InputError.tsx

```diff
@@ -0,0 +1,21 @@
+import React from "react";
+
+const InputError: React.FC<{ isVisible: boolean; error: string }> = ({
+  error,
+  isVisible,
+}) => {
+  return isVisible ? (
+    <p
+      data-h2-display="b(inline-block)"
+      data-h2-radius="b(s)"
+      data-h2-bg-color="b(red[.1])"
+      data-h2-padding="b(all, xs)"
+      data-h2-color="b(red)"
+      role="alert"
+    >
```

Comment:
might be a good idea to add `data-h2-font-*` properties such as `data-h2-font-family="b(sans)"` to match the design, unless styles will be inherited from a planned higher component?

Reply:
I assume the inputs should use the same font as the rest of the app?

However, I did just notice that the designs seem to use a different font size for the label and error.

Reply:
I added default font family in #220